chain/x/zoracle/client/cli/tx.go

```diff
@@ -519,3 +523,113 @@ $ %s tx zoracle edit-oracle-script 1 --name eth-price --description "Oracle scri
 
 	return cmd
 }
+
+// GetCmdAddOracleAddress implements the adding of oracle address command handler.
+func GetCmdAddOracleAddress(cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "add-oracle-address (--validator [validator]) (--reporter [reporter])",
```

Comment:
Let's make it positional arguments. It's just two arguments and I think it's the standard.